austation/code/game/objects/items/deepbagofholding.dm

```diff
@@ -0,0 +1,239 @@
+//  Deep bag of holding. Still unsure as to how it will be made.
+
+/obj/item/deepbackpack
+	name = "deep bag of holding"
+	desc = "A bluespace bag, with a pocket dimension large enough to live in. It's huge!"
+	// Alternative descriptions: "Enslaved Tardis", "It's bigger on the inside", "Mobile ERP den", "Mobile dormitory", "Mini drug lab", e.t.c
```

Comment:
should have gone for mobile erp den in my honest opinion.

Reply:
Too direct. Plus, I figured that wouldn't get elon's vote.

Reply:
:clap: not :clap: an :clap: erp :clap: server :clap: